### ISSUE TITLE
Missing icons

### ISSUE THREAD

**Issue by BAndiT1983** [apertus-open-source-cinema/web-remote#3]
Some icons seem to be missing, like "battery_unknown" and "check_circle". They appear as text.

**Comment by Mandrake0** [comment]
yes i know this, it's because this material icons set is >3 years old and for the new ones a build setup has to be made. there is a list what icons are in files: https://github.com/google/material-design-icons/blob/master/iconfont/codepoints
